.devcontainer/docker-compose-vscode.yml

```diff
@@ -72,8 +72,9 @@ services:
       DOCKER_HOST_IP:  127.0.0.1
       KAFKA_ADVERTISED_HOST_NAME:  127.0.0.1
       KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
-      KAFKA_CREATE_TOPICS: "time_sync:1:1"
+      KAFKA_CREATE_TOPICS: "time_sync:1:1,modified_spat:1:1,v2xhub_scheduling_plan_sub:1:1,v2xhub_ssm_sub:1:1"
       KAFKA_LOG_DIRS: "/kafka/kafka-logs"
+      KAFKA_OFFSETS_RETENTION_MINUTES: 1
```

Comment:
I assume this is a configuration to set how long Kafka will maintain offsets for a consumer group. Is that correct? Can you explain why adding this configuration is important?